File: app/src/main/java/org/lantern/app/model/Stats.java
package org.lantern.app.model;

public class Stats {
    private String city, country, countryCode;
    private long httpsUpgrades;
    private long adsBlocked;

    public Stats(String city, String country, String countryCode, long httpsUpgrades, long adsBlocked) {

        this.city = city;
        this.country = country;
        this.countryCode = countryCode;
        this.httpsUpgrades = httpsUpgrades;
        this.adsBlocked = adsBlocked;
    }

    public String getCountry() {
        return country;
    }

    public String getCity() {
        return city;
    }

    public String getCountryCode() {
        return countryCode;
    }

    public String getHTTPSUpgrades() {
        return String.valueOf(httpsUpgrades);
    }

    public String getAdsBlocked() {
        return String.valueOf(adsBlocked);
    }
}

File: android/app/src/main/kotlin/org/getlantern/lantern/model/Stats.kt
package org.getlantern.lantern.model

data class Stats(
  val city: String = "",
  val country: String = "",
  val countryCode: String = "",
  val httpsUpgrades: Long = 0,
  val adsBlocked: Long = 0,
  val hasSucceedingProxy:Boolean = false
) {

}